atd-vze/src/views/Crashes/crashDataMap.js

```diff
@@ -96,108 +94,86 @@ export const createCrashDataMap = isTempRecord => {
           label: "Crash Severity",
-          editable: true,
-          uiType: "select",
-          lookupOptions: "atd_txdot__injry_sev_lkp",
-          lookupPrefix: "injry_sev",
-        },
-        vz_fatality_count: {
-          label: "ATD Fatality Count",
-          editable: true,
-          uiType: "text",
-        },
-        death_cnt: {
-          label: "CRIS Death Count",
+          relationshipName: "crash_injury_metrics_view",
           editable: false,
+          uiType: "select",
+          lookupOptions: "injry_sev_lkp",
         },
-        apd_confirmed_death_count: {
-          label: "APD Death Count",
-          editable: true,
-          uiType: "text",
-          secondaryFieldUpdate: { apd_human_update: "Y" },
-        },
-        apd_human_update: {
-          label: "Manually Edited?",
-          editable: false,
-          lookupOptions: "atd_txdot__y_n_lkp",
-          lookupPrefix: "y_n",
-        },
-        law_enforcement_num: {
-          label: "Law Enforcement Number",
-          editable: true,
-          uiType: "text",
-        },
-      },
-    },
-    {
-      title: "Injuries",
-      fields: {
-        non_injry_cnt: {
+        non_injry_count: {
           label: "Not Injured Count",
+          relationshipName: "crash_injury_metrics_view",
           editable: false,
         },
-        nonincap_injry_cnt: {
+        nonincap_injry_count: {
           label: "Non-incapacitating Injury Count",
+          relationshipName: "crash_injury_metrics_view",
           editable: false,
         },
-        poss_injry_cnt: {
+        poss_injry_count: {
           label: "Possible Injury Count",
+          relationshipName: "crash_injury_metrics_view",
           editable: false,
         },
-        sus_serious_injry_cnt: {
+        sus_serious_injry_count: {
           label: "Suspected Serious Injury Count",
-          editable: true,
+          relationshipName: "crash_injury_metrics_view",
+          editable: false,
           uiType: "text",
         },
-        tot_injry_cnt: {
-          label: "Total Injury Count",
-          editable: true,
+        // tot_injry_cnt: {
+        //   label: "Total Injury Count",
+        //   relationshipName: "crash_injury_metrics_view",
+        //   editable: false,
+        //   uiType: "text",
+        // },
+        unkn_injry_count: {
+          label: "Unknown Injury Count",
+          relationshipName: "crash_injury_metrics_view",
+          editable: false,
+        },
+        vz_fatality_count: {
+          label: "ATD Fatality Count",
+          relationshipName: "crash_injury_metrics_view",
+          editable: false,
           uiType: "text",
         },
-        unkn_injry_cnt: {
-          label: "Unknown Injury Count",
+        cris_fatality_count: {
+          label: "CRIS Death Count",
+          relationshipName: "crash_injury_metrics_view",
           editable: false,
         },
+        // apd_confirmed_death_count: {
```

Comment:
@johnclary is this still a field we want? i dont think we have it in the new data model yet

Reply:
ah yes—this is now `law_enf_fatality_count` in the metrics view. Would you change the label to "Law Enforcement Fatality Count" ?